cooperator/tests/cooperator_test_mixin.py

```diff
@@ -0,0 +1,145 @@
+
+        receivable_account_type = cls.env.ref("account.data_account_type_receivable")
+        equity_account_type = cls.env.ref("account.data_account_type_equity")
+        cooperator_account = account_model.create(
+            {
+                "name": "Cooperator Test",
+                "code": "416109",
+                "user_type_id": receivable_account_type.id,
+                "reconcile": True,
+            }
+        )
+        cls.company.property_cooperator_account = cooperator_account
+        cls.equity_account = account_model.create(
+            {
+                "name": "Equity Test ",
+                "code": "100919",
+                "user_type_id": equity_account_type.id,
+                "reconcile": True,
+            }
+        )
+        subscription_journal_sequence = cls.env.ref(
+            "cooperator.sequence_subscription_journal"
+        )
+        cls.subscription_journal = cls.env["account.journal"].create(
+            {
+                "name": "Subscriptions Test",
+                "code": "SUBJT",
+                "type": "sale",
+                "sequence_id": subscription_journal_sequence.id,
+            }
+        )
+
+        cls.share_x = cls.env["product.product"].create(
+            {
+                "name": "Share X - Founder",
+                "short_name": "Part X",
+                "is_share": True,
+                "by_individual": True,
+                "by_company": True,
+                "list_price": 50,
+            }
+        )
+        cls.share_y = cls.env["product.product"].create(
+            {
+                "name": "Share Y - Worker",
+                "short_name": "Part Y",
+                "is_share": True,
+                "default_share_product": True,
+                "by_individual": True,
+                "by_company": False,
+                "list_price": 25,
+            }
+        )
+        cls.subscription_request_1 = cls.env["subscription.request"].create(
+            {
+                "firstname": "John",
+                "lastname": "Doe",
+                "email": "john@test.com",
+                "address": "Cooperation Street",
+                "zip_code": "1111",
+                "city": "Brussels",
+                "lang": "en_US",
+                "country_id": cls.env.ref("base.be").id,
+                "date": datetime.now() - timedelta(days=12),
+                "source": "manual",
+                "ordered_parts": 3,
+                "share_product_id": cls.share_y.id,
+                "data_policy_approved": True,
+                "internal_rules_approved": True,
+                "financial_risk_approved": True,
+                "generic_rules_approved": True,
+                "gender": "male",
+                "iban": "09898765454",
+                "birthdate": date(1990, 9, 21),
+                "skip_iban_control": True,
+            }
+        )
+
+    @classmethod
+    def _ensure_account_property(cls, model, property_name):
```

Comment:
“ensure” lacks something: ensure that what? rename to `_ensure_account_property_is_set()`? or more simply `_set_account_property()`?

Reply:
I was inspired by bits of code from the odoo core but I think your naming is better 👍

Reply:
I kept `_ensure_account_property_is_set` because 
- I don't set the property if it is already set
- I set an arbitrary account